packages/updatePrescriptionStatus/src/updatePrescriptionStatus.ts

```diff
@@ -2,194 +2,156 @@
 import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda"
 import {Logger} from "@aws-lambda-powertools/logger"
 import {injectLambdaContext} from "@aws-lambda-powertools/logger/middleware"
-import {DynamoDBClient, PutItemCommand} from "@aws-sdk/client-dynamodb"
-import {marshall} from "@aws-sdk/util-dynamodb"
 import middy from "@middy/core"
 import inputOutputLogger from "@middy/input-output-logger"
 import errorHandler from "@nhs/fhir-middy-error-handler"
+import {Bundle, BundleEntry, Task} from "fhir/r4"
+
+import {transactionBundle, validateEntry} from "./validation/content"
+import {
+  accepted,
+  badRequest,
+  bundleWrap,
+  createSuccessResponseEntries,
+  serverError
+} from "./utils/responses"
+import {persistDataItems} from "./utils/databaseClient"
 
 const logger = new Logger({serviceName: "updatePrescriptionStatus"})
-const client = new DynamoDBClient({region: "eu-west-2"})
-const tableName = process.env.TABLE_NAME
-
-interface DynamoDBItem {
-  RequestID: string | undefined;
-  PrescriptionID: string;
-  PatientNHSNumber: string;
-  PharmacyODSCode: string;
-  TaskID: string;
-  LineItemID: string;
-  TerminalStatus: string;
-  RequestMessage: any;
+
+export interface DataItem {
+  LastModified: string
+  LineItemID: string
+  PatientNHSNumber: string
+  PharmacyODSCode: string
+  PrescriptionID: string
+  RequestID: string
+  RequestMessage: Task
+  Status: string
+  TaskID: string
+  TerminalStatus: string
```

Comment:
I'd rather not duplicate all this data, I think the only items required at the top level are PrescriptionID, LineItemID, and TaskID. Those are the ones that should go in the partition and sort keys. Everything else can be pulled from the request message.

Reply:
Changed according to the commit made in our call.